scripts/run.sh

```diff
@@ -22,6 +22,18 @@ function cleanup {
 # NATS
 ###
 gnatsd &
+counter=1
+until nc -zv localhost 4222 1>/dev/null 2>&1; 
+do
+    sleep 0.5
+    ((counter++))
+    if [ ${counter} -gt 20 ]
+    then
+        echo "gnatsd failed, exiting"
```

Comment:
I would change the message to "gnatsd failed to start in 10 seconds, exiting" so that the user knows what the problem was and maybe raise the timeout if needed.

Reply:
10 sec is too much, NATS usually starts in a friction of second. I would opt for 3-5 sec, not more.